README.md

```diff
@@ -2,29 +2,30 @@
 [![Build Status](https://travis-ci.org/spacejam/rust-crdt.svg?branch=master)](https://travis-ci.org/spacejam/rust-crdt)
 [![crates.io](http://meritbadge.herokuapp.com/crdts)](https://crates.io/crates/crdts)
 
-Thoroughly tested serializable practical CRDT's ported from riak_dt.
-
+A family of thoroughly tested CRDT's.
 
 [documentation](https://docs.rs/crdts/1.2.11/crdts/)
 
+- **VClock**: Vector clock. Typically used to track the causal history
+- **ORSWOT**: Observed-Remove Set Without Tombstones. An add-biased set
+- **Map**: Add biased Map with reset-remove semantics. Map values are also CRDT's
+- **MVReg**: Multi-Value Register. Holds a value, concurrent edits are delt with by storing both.
+- **LWWReg**: Last-Write-Wins Register. Holds a value, concurrent edits are resolved by keeping the value with the largest counter.
+- **GCounter**: Grow-only Counter. A counter that only goes up.
+- **PNCounter**: Pos/Neg Counter. A counter that can go up and down.
+- **GSet**: Grow-only set. A set that only grows
 
-- [x] Vector Clock
-- [x] ORSWOT
-- [x] LWW Register
-- [x] G-Counter
-- [ ] Top-K Set
-- [ ] Map
-- [ ] G-Set
-- [ ] OR-Set
-- [x] PN-Counter
-- [ ] EM-Counter
 
+#### Things we still need to build (PR's welcome)
```

Comment:
While I understand both items, it is a bit less clear to me what are your wishes and which kind of API you envision. Would you mind opening a ticket for each and then linking them here?

Reply:
I'll move these to issues, I'm not sure it needs to stay in the README, maybe I'll bring it back..